sdk/src/error.rs

```diff
@@ -0,0 +1,25 @@
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("invalid subnet id")]
+    InvalidID,
+    #[error("invalid IPC address")]
+    InvalidIPCAddr,
+    #[error("fvm shared address error")]
+    FVMAddressError(fvm_shared::address::Error),
```

Comment:
This error is duplicated in `gateway/types.rs`. Is this a mistake?

Reply:
you are right, no need now, very sharp!